### ISSUE TITLE
Broken link in CI/CD Workshop

### ISSUE THREAD

**Issue by pendsley** [pactflow/docs.pactflow.io#8]
There's a broken link on the CI/CD Workshop

https://docs.pactflow.io/docs/workshops/ci-cd/
The GOALS button takes you to https://docs.pactflow.io/docs/workshops/ci-cd/set-up-ci/index (Which falls back to docs.pactflow.io)

I think it's supposed to be this instead: https://docs.pactflow.io/docs/workshops/ci-cd/set-up-ci/

**Comment by bethesque** [comment]
Interesting. That link is generated by Docusaurus based on the configuration here: https://github.com/pactflow/docs.pactflow.io/blob/master/website/sidebars.json#L23 I wonder if we've configured it incorrectly, or if there's a bug. I'll have a look into it - thank you for raising it.

**Comment by bethesque** [comment]
Ok, I've identified the problem. We've got "clean path" turned on, which allows `/foo.html` to be accessed via `/foo` by copying `/foo.html` to `/foo/index.html` when the site is generated. For some reason, it has an explicit check not to do this if the file is called `index.html`

https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-1.x/lib/server/generate.js#L50

The "next" link that is generated at the bottom of the page does not know about this rule though, so it's generating the the url `/index` which doesn't match `index.html`, so it's considering it a 404, and returning the site index page.

A quick hack would be to copy `index.html` to `index/index.html` when we build the site. The more correct fix would be to correct the link, however, they're currently working on docusaurus v2, so they may not be doing bug fixes for v1.
